src/tours/Tour/styles.scss

```diff
@@ -0,0 +1,55 @@
+.interactive-tour {
```

Comment:
if possible we should use css modules for this.  it may be precluded by use of react-tour library but we can discuss